MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+include LICENSE
+include *.md
+include setup.cfg
+recursive-include pythoncz *.py
```

Comment:
`*.py` se tu podle mě vyjmenovávat nemusí :)

Pak kdyžtak můžu poslat PR s revizí, co zde má nebo nemusí být. Při instalaci přes `pip install -e` toto není moc důležité. Jen nevím, jak to vůbec mohlo fungovat na produkci, dokud tento soubor nebyl :)

Reply:
Díval jsem se [sem](https://github.com/django/django/blob/master/MANIFEST.in), ale byl jsem líný zjišťovat, co znamená `graft`.